src/mainview.h

```diff
@@ -18,6 +22,7 @@ namespace ui
 	class CMainView : public QDialog
 	{
 		friend class CEditConfig;
+		friend class CModManager;
```

Comment:
The reason they have this is because I don't feel comfortable giving out m_InstallDir (held by CMainView) to anything other than those two. I guess a simple getter would fix this.